### ISSUE TITLE
Safari 13 compatibility?

### ISSUE THREAD

**Issue by matthewkidd** [phallstrom/SafariFontScope#11]
Any chance of updating this extension to run on macOS Catalina? I didn't fully appreciate how crucial this extension is to my daily browsing until Apple took it away!

**Comment by phallstrom** [comment]
Is it Safari 13 or Catalina that is the issue?  I'll get there eventually, but don't plan to install the Catalina preview...

**Comment by sfhosley** [comment]
@phallstrom It's Safari 13. Just got this message when I updated.

<img width="408" alt="fontscope-safari_13" src="https://user-images.githubusercontent.com/5246930/65273417-b9707900-dae6-11e9-8219-1d899cb381c9.png">

**Comment by elemis** [comment]
Will there be any updates for Safari 13? I don't know what will I do without this extension.

**Comment by git-jannis** [comment]
Agree, I can not live without this extension. Please update it soon <3

**Comment by git-jannis** [comment]
Is there any chance for an update this year? I would pay for it …

**Comment by phallstrom** [comment]
I dunno. I gave it a shot once, but now it requires a legit Apple app plus extension.  Whole new world for me... without good docs that I've been able to find :/  I'd like to, but it might be awhile.
